tests/test_interface.rb

```diff
@@ -1485,4 +1485,227 @@ def test_ipv4_pim_sparse_mode
     i.ipv4_pim_sparse_mode = i.default_ipv4_pim_sparse_mode
     assert_equal(i.default_ipv4_pim_sparse_mode, i.ipv4_pim_sparse_mode)
   end
+
+  def test_storm_control_broadcast
```

Comment:
So this is a bit beyond the scope of this PR but I think it's small enough and will add enough value that we should go ahead and do it.

The tests in this file enable quite a few features but never turn them off which can cause problems for other tests.  Chris added some logic a few days ago that allows you to determine if the last test case being run.

https://github.com/cisco/cisco-network-node-utils/commit/9f13ed247b75b2561b1b91daad5ab92f59edd566

I suggest you do the following.

- Turn of all features on the box (n3k, n5/6k, n7k, n9k) and run the tests in this file to see all of the features that are still enabled when the test complete.
- Create a method to disable all of the features and call it in the `teardown` method if it's the last test being run.